doc/python/axes.md

```diff
@@ -401,7 +399,7 @@ fig = px.scatter(df, x="total_bill", y="tip", color="sex")
 
 
 fig.update_xaxes(minor=dict(ticklen=6, tickcolor="black", showgrid=True))
-fig.update_yaxes(minor=dict(ticks="inside"))
+fig.update_yaxes(minor_ticks="inside")
```

Comment:
when updating a single attribute or two, I try to favour 'magic underscores' like this, to remind folks this exists